.travis.yml

```diff
@@ -3,7 +3,7 @@ dist: bionic
 sudo: false
 
 mono: none
-dotnet: 2.2.203
+dotnet: 3.1.300
```

Comment:
Do we use APIs of .NET Core 3.1? If not, I'd like to keep the version as low as possible. This way, more people can use this library.
If we're using .NET Core 3.1 features, it would be more than valid (since it is LTS).

Reply:
We can stay on older version, I just prefer to work with newer versions

Reply:
dotnet 2.2 dropped out of support in dec 2019 so I guess upgrading to 3.1 LTS is a good idea 

https://dotnet.microsoft.com/platform/support/policy/dotnet-core


Version | Original Release Date | Latest Patch Version | Patch Release Date | Support Level | End of Support
-- | -- | -- | -- | -- | --
.NET Core 3.1 | December 3, 2019 | 3.1.6 | July 14, 2020 | LTS | December 3, 2022
.NET Core 3.0 | September 23, 2019 | 3.0.3 | February 18, 2020 | EOL | March 3, 2020
.NET Core 2.2 | December 4, 2018 | 2.2.8 | November 19, 2019 | EOL | December 23, 2019
.NET Core 2.1 | May 30, 2018 | 2.1.20 | July 14, 2020 | LTS | August 21, 2021

Reply:
We already do.

Reply:
missed that sry :)